api/database/dbOperations.ts

```diff
@@ -96,4 +96,117 @@ const fetchData = async (
   return { mainData, columnsData, metadata };
 };
 
-export default fetchData;
+interface ViewConfig {
+  VIEWS: string;
+  ALERT_RESOURCES: string;
+  EMBED_MEDIA: string;
+  FILTER_BY_COLUMN: string;
+  FILTER_OUT_VALUES_FROM_COLUMN: string;
+  FRONT_END_FILTERING: string;
+  FRONT_END_FILTER_COLUMN: string;
+  MAPBOX_STYLE: string;
+  MAPBOX_PROJECTION: string;
+  MAPBOX_CENTER_LATITUDE: string;
+  MAPBOX_CENTER_LONGITUDE: string;
+  MAPBOX_ZOOM: string;
+  MAPBOX_PITCH: string;
+  MAPBOX_BEARING: string;
+  MAPBOX_3D: string;
+  MAPEO_TABLE: string;
+  MAPEO_CATEGORY_IDS: string;
+  MAP_LEGEND_LAYER_IDS: string;
+  MEDIA_BASE_PATH: string;
+  MEDIA_BASE_PATH_ALERTS: string;
+  LOGO_URL: string;
+  PLANET_API_KEY: string;
+  UNWANTED_COLUMNS?: string;
+  UNWANTED_SUBSTRINGS?: string;
+}
+
+interface Views {
+  [key: string]: ViewConfig;
+}
+
+export const fetchConfig = async (
+  db: any,
+  isSQLite: boolean | undefined,
+): Promise<Views> => {
+  // Create the config table if it does not exist
+  const createConfigTable = `CREATE TABLE IF NOT EXISTS config (
+         table_name TEXT PRIMARY KEY,
+         config TEXT
```

Comment:
These names feel extremely elusive, like there might be many things in an app called "config", so which does this store?  Also having `table_name TEXT` as a primary key, I am really struggling to understand what is stored in this table.

I wonder what this table and columns would be named if you went one  notch "more specific"

Reply:
Thanks for this! Yes, I will improve the naming here before merging.